### DISCUSSION TITLE
Impressive!

### DISCUSSION THREAD

**Discussion by yezzer** [lambtor/ArduinoTricorder#2] [General]
Just wanted to say, this is super cool. I'd just got a thermal camera + SPI LCD working on a Pi, and thought maybe i could make some kind of tricorder - had a quick google and found this. Looks great! 

Top marks for all the details you've provided.

I'll be making something else then 😄

**Comment by lambtor** [comment]
Thanks! I know a few of the parts aren't easy prints, and the last step to assemble the body face and back shell aren't easy.  You're free to remix the shell to suit your needs, of course, as the top board mounting points are meant for the adafruit sense.  If you're comfortable coding in Arduino, the sketch should hopefully be adaptable enough to suit your needs.
